### ISSUE TITLE
Unmount external drives when switching to battery power is not working

### ISSUE THREAD

**Issue by Ati7479** [fcpxhacks/fcpxhacks#3335]
Within Command Post, Preferences, Finder,  Drive Management, upon disconnecting my MacBook from power, or connecting to power, external drives are not mounted or unmounted. I do get a notification that it is, but drives are not mounted or unmounted. 
Latest version of Command Post 1.4.26

<img width="1242" alt="Screenshot 2024-04-26 at 10 45 10 AM" src="https://github.com/CommandPost/CommandPost/assets/168205722/59a39d9d-63c5-4a98-87c2-0295093e995c">
<img width="280" alt="Screenshot 2024-04-26 at 10 45 15 AM" src="https://github.com/CommandPost/CommandPost/assets/168205722/a5fd9e54-3481-4d49-ab4e-5181942aa8ad">
<img width="1000" alt="Screenshot 2024-04-26 at 10 46 27 AM" src="https://github.com/CommandPost/CommandPost/assets/168205722/a9445ffd-29c1-4c08-9f03-8342f2ba7024">
<img width="1454" alt="Screenshot 2024-04-26 at 10 49 09 AM" src="https://github.com/CommandPost/CommandPost/assets/168205722/d14a0506-77e7-4783-b1be-2d6fdc51271f">
<img width="1467" alt="Screenshot 2024-04-26 at 10 49 16 AM" src="https://github.com/CommandPost/CommandPost/assets/168205722/537957f8-9aba-4440-a929-d88f41cb4dd8">

**Comment by latenitefilms** [comment]
@randomeizer - Do you still use this feature?

**Comment by randomeizer** [comment]
Occasionally, but I have noted that while my regular external drive is unmounted correctly, Time Machine drives do not. I meant to look into it sometime.

**Comment by Ati7479** [comment]
It would be nice to use this if it was working properly, I have several computers and drives, I have to manually eject them, no big deal but it would be convenient if this feature worked. Perhaps my dongle is the issue? Drives are not directly plugged into the MacBook.

**Comment by randomeizer** [comment]
Yeah, I'll have a play and see if something has changed in recent versions of the OS. What version of macOS are you on?

**Comment by randomeizer** [comment]
Never mind, it's Sonoma. I'm on that also.

**Comment by latenitefilms** [comment]
@randomeizer - Did you ever look into this?

**Comment by randomeizer** [comment]
I didn't. I'll see if I can get some time tomorrow.
